### PR TITLE
Make remote playing available

### PR DESCRIPTION
**Pull request by zidder**

### CODE REVIEW

File: game/player.py [hunk 1]
```diff
@@ -1,16 +1,37 @@
...
+        self.args_kwargs = (args, kwargs)
+
     def decide(self):
-        self.socket.get_decision()
+        # WARNING: don't use pickle in production codes
+        return pickle.loads(self.conn.recv(256))
 
     def observe(self, sight):
-        self.socket.send_sight(sight)
+        self.conn.sendall(pickle.dumps(sight))
```
Comment (fisheeve) [inline game/player.py:20]: self.conn is not defined, is it?

**Review by fisheeve — COMMENTED** [review]
Add pickle and socket to requirements.txt

**Comment by zidder** [comment]
> Add pickle and socket to requirements.txt

Those are python builtins.

File: game/player.py (hunk 1 above)
Comment (zidder) [reply game/player.py:20]: it is after `connect`

**Pull request merged**